wiki/Tournaments/Template/en.md

```diff
@@ -0,0 +1,150 @@
+---
+tags:
+- {ABBR}
+---
+
+<!--
+Find and replace those tags listed below with proper information, then delete this comment.
+{TOURNAMENT_NAME} = tournament name
+{MODE} = game mode
+{ABBR} = abbreviation of the tournament name followed by its iteration/year occurence i.e. MWC 4K 2018
+{ORGANISER} = organiser of the tournament
+{NTH} = installment ordinal number
+{TOURNAMENT_SERIES} = tournament name without its iteration suffix i.e. osu!mania 4K World Cup
+{DRAW_TIME} = Live Drawings (Livestream) time for the tournament
+{TOPIC_NUM} = forum thread number
+{STATS_LINK} = link to overall statistics sheet (it should be a Google Drive link)
+You will still need to fill other items such as tables, etc.
+Use .png as a default format for images, since .jpg can't handle transparency.
+Alongside this comment, remember to delete all comments across the article after you finish.
+-->
+
+# {TOURNAMENT_NAME}
+
+![{ABBR} Logo](img/logo.png)
+
+The **{TOURNAMENT_NAME}** (***{ABBR}***) is a country-based {MODE} tournament hosted by the {ORGANISER}. It is the {NTH} installment of the {TOURNAMENT_SERIES}.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-01-01/2018-01-28 |
+| Live Drawings | 2018-01-29 {DRAW_TIME} |
+| Group Stage |  |
+| Round of 16 |  |
+| Quarterfinals |  |
+| Semifinals |  |
+| Finals - Week 1 |  |
+| Finals - Week 2 |  |
+
+<!-- Use YYYY-MM-DD Format for dates i.e. 2018-01-01
+Use (HH:MM) format for {DRAW_TIME} i.e. (12:00 UTC)
+ -->
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for {TOURNAMENT_NAME} winners. It is possible that these items change with every installment of the {TOURNAMENT_NAME}.
```

Comment:
`every installment of the {TOURNAMENT_NAME}` -> `every installment of the {TOURNAMENT_SERIES}`